### PR TITLE
Clean up outdated extensions etc.

### PR DESCRIPTION
**Pull request by emlys**
* Got rid of the `numfig` extension. The same functionality (which we only use in two files) is now available just by setting `numfig = True` in the conf.py. 
* Removed the `json` extension, which as far as I can tell we weren't using anyway
* Got rid of all references to the primer. It's a neat idea, and maybe sometime in the future there will be a use for this, or something like it. If so all the `.. primer` labels are here in dd0ee6c ! But IMO it's not worth keeping that Makefile and conf.py file around, since it was starting to accumulate deprecated things.
* Updated all broken/redirected hyperlinks
* Got rid of MS Word docs and `supplements/`, since they seem to only be there for historical reasons, and will remain available in git history.
* Cleaned up the Makefile and removed `doctest` target, since we have not `.. doctest` blocks anyway.

How does the ReadTheDocs build work? Maybe it does somehow have access to a `json` extension that I don't know about...

### CODE REVIEW

File: source/habitat_quality.rst [hunk 1]
```diff
@@ -1,5 +1,3 @@
-.. primer
-
 .. _habitat_quality:
 
 .. |addbutt| image:: ./shared_images/addbutt.png
```
Comment (emlys) [inline source/habitat_quality.rst:3]: 🤣

**Comment by dcdenu4** [comment]
Hey @emlys can you remind me if this is going to stay as Draft until after the 3.9 release? I kind of remember that from the coffee call. Just wanted to make sure I wasn't ignoring this.

**Comment by emlys** [comment]
> Hey @emlys can you remind me if this is going to stay as Draft until after the 3.9 release? I kind of remember that from the coffee call. Just wanted to make sure I wasn't ignoring this.

Yep it's good to wait until after the release to merge! You're welcome to review it now if you want though :)

File: source/carbonstorage.rst [hunk 2]
```diff
@@ -178,11 +175,10 @@ Land use/land cover
 
 Global land use data is available from:
 
- *  NASA: https://lpdaac.usgs.gov/dataset_discovery/modis/modis_products_table/mcd12q1 (MODIS multi-year global landcover data provided in several classifications)
- *  The European Space Agency: https://www.esa-landcover-cci.org (Three global maps for the 2000, 2005 and 2010 epochs) 
- *  The University of Maryland�s Global Land Cover Facility: http://glcf.umd.edu/data/landcover/ (data available in 1 degree, 8km and 1km resolutions).
```
Comment (dcdenu4) [inline source/carbonstorage.rst:183]: Is there a reason this link was deleted?

File: source/habitat_risk_assessment.rst [hunk 3]
```diff
@@ -77,28 +75,26 @@ The risk of human activities to habitats or species is modeled in five steps.
...
 
 **Step 2.** The second step combines the exposure and response values to produce a risk value for each stressor-habitat combination in each grid cell. There are two options for risk calculation.
 
-For Euclidean Risk calculation, risk to habitat *j* caused by stressor *k* in each location (i.e. cell) *l* is calculated as the Euclidean distance from the origin in the exposure-consequence space, where average exposure (:ref:`eq1`) is on one axis and the average consequence score (:ref:`eq2`) is on the other.
+For Euclidean Risk calculation, risk to habitat *j* caused by stressor *k* in each location (i.e. cell) *l* is calculated as the Euclidean distance from the origin in the exposure-consequence space, where average exposure (:eq:`exposure`) is on one axis and the average consequence score (:eq:`consequence`) is on the other.
 
 .. math:: R_{jkl} = \sqrt{(E_{jkl}-1)^2+(C_{jkl}-1)^2}
-   :label: eq3
```
Comment (dcdenu4) [inline source/habitat_risk_assessment.rst:92]: Is there a reason for removing the labels here? I'm just wondering if they might be referenced in the HRA source code? I believe it is something we've done in the past, though that kind of coupling is probably not ideal given the need to upkeep both.

File: source/invest_api.rst [hunk 4]
```diff
@@ -17,11 +15,8 @@ Summary
...
 
-While there is a user interface for all the InVEST models, we also provide a python application programming interface (API) for more advanced usage of the models.  Full documentation for the ``natcap.invest`` python package is available here: http://invest.readthedocs.io/en/latest/
-
-Additionally, our core geoprocessing routines have been open sourced as a separate python package, ``pygeoprocessing``.  Documentation for ``pygeoprocessing`` is available here: http://pythonhosted.org/pygeoprocessing/
-
-This documentation will be updated in more detail in the future.  Programming support is also available on our user forums: http://forums.naturalcapitalproject.org/.
+While there is a user interface for all the InVEST models, we also provide a python application programming interface (API) for more advanced usage of the models.  Full documentation for the ``natcap.invest`` python package is available here: https://invest.readthedocs.io/en/latest/
 
-.. primerend
+Additionally, our core geoprocessing routines have been open sourced as a separate python package, ``pygeoprocessing``, available here: https://github.com/natcap/pygeoprocessing.
```
Comment (dcdenu4) [inline source/invest_api.rst:20]: I think James might have recently done some API docs for Pygeoprocessing?

File: source/sdr.rst [hunk 5]
```diff
@@ -465,11 +460,10 @@ A key component for all water models is a spatially continuous land use/land cov
 
 Global land use data is available from:
 
- *  NASA: https://lpdaac.usgs.gov/dataset_discovery/modis/modis_products_table/mcd12q1 (MODIS multi-year global landcover data provided in several classifications)
- *  The European Space Agency: https://www.esa-landcover-cci.org (Three global maps for the 2000, 2005 and 2010 epochs)
- *  The University of Maryland's Global Land Cover Facility: http://glcf.umd.edu/data/landcover/ (data available in 1 degree, 8km and 1km resolutions).
```
Comment (dcdenu4) [inline source/sdr.rst:470]: Is this deprecated?

File: source/wave_energy.rst [hunk 6]
```diff
@@ -73,19 +70,19 @@ Potential Wave Power Resource Assessment
...
 where, :math:`P_n` is wave power (kW/m), :math:`\rho` is sea water density (1,028 kg m\ :sup:`-3`), :math:`g` is gravitational acceleration (9.8 m s\ :sup:`-2`), :math:`H_s` is significant wave height (m), and :math:`C_g` is wave group velocity (m s\ :sup:`-1`) as a function of wave energy period, :math:`T_e` (sec), and water depth :math:`h` (m) (Cornett 2008). :math:`C_g` can be estimated as
 
 .. math:: C_g = { {\left(1 + {{2kh}\over \sinh(2kh)}\right) \sqrt{{g\over k} \tanh(kh)}}\over 2 }
-   :label: eq2
+   :label: wave_group_velocity
 
 where the wave number :math:`k` is calculated using a dispersion relationship expressed as a function of wave frequency (:math:`w = 2\pi / T_e`) and water depth :math:`h`:
 
 .. math:: w^2 = {gk * \tanh(kh)}
-   :label: eq3
```
Comment (dcdenu4) [inline source/wave_energy.rst:86]: Again wondering if these are referenced in the source code.

File: source/wind_energy.rst [hunk 7]
```diff
@@ -70,21 +68,20 @@ where, :math:`k` and :math:`\lambda` are the shape and scale factors, respective
...
+Wind power density is calculated at the hub height :math:`Z` (m) of a wind turbine (:numref:`weibull-fig`), which means all variables in :eq:`wind_power_density` and :eq:`weibull_dist` need to be converted into the appropriate value at hub height. Mean air density :math:`\rho` was estimated as :math:`\rho=1.225-(1.194\cdot 10^{-4})Z`, which approximates the U.S. Standard Atmosphere profile for air density (National Oceanic and Atmospheric Administration, 1976). We applied the wind profile power law to estimate wind speed (:math:`V`) at hub height :math:`Z` (Elliott et al., 1986).
 
 .. math:: \frac{V}{V_r} = \left(\frac{Z}{Z_r}\right)^\alpha
-   :label: eq3
```
Comment (dcdenu4) [inline source/wind_energy.rst:76]: I also wonder if keeping the equation number labels is more conforming to scientific papers as well. Might be more the "norm"

**Review by dcdenu4 — COMMENTED** [review]
Hey @emlys , thanks for taking this on. Clearly needed a clean up. I left a few scattered comments about. It also looks like there are some merge conflicts currently.

File: source/carbonstorage.rst (hunk 2 above)
Comment (emlys) [reply source/carbonstorage.rst:183]: The site no longer exists, and it seems the whole project was discontinued. I couldn't find the same dataset anywhere.

File: source/habitat_risk_assessment.rst (hunk 3 above)
Comment (emlys) [reply source/habitat_risk_assessment.rst:92]: Having multiple `eq1`s in different model files raises a warning. Each label is supposed to be unique. Giving them more descriptive labels is kinda tedious so I just deleted the ones that aren't referenced anywhere. And the labels that are actually displayed seem to be  different. E.g. the equation labeled `:label: cbc_emissions` in https://storage.googleapis.com/releases.naturalcapitalproject.org/invest-userguide/latest/coastal_blue_carbon.html#the-model shows up with a label (5) [here](https://storage.googleapis.com/releases.naturalcapitalproject.org/invest-userguide/latest/coastal_blue_carbon.html#the-model). I think they always are numbered from the top of the page.

File: source/invest_api.rst (hunk 4 above)
Comment (emlys) [reply source/invest_api.rst]: good point! Added the link to that

File: source/wave_energy.rst (hunk 6 above)
Comment (emlys) [reply source/wave_energy.rst:86]: Same as above, this label shouldn't affect how it's labeled in the built docs.

**Comment by emlys** [comment]
Thanks for the review @dcdenu4 ! Addressed your comments above.

**Comment by emlys** [comment]
Sorry @dcdenu4 I added some more changes! They're all in a7331cf. Updating some deprecated options and removed the unused config template stuff (a lot of it is now deprecated). Also took out the makefile commands that we don't generally use, and the latex config options (was causing a bunch of warnings, and I don't think it's worth maintaining if we're not using latex).

File: source/habitat_risk_assessment.rst (hunk 3 above)
Comment (dcdenu4) [reply source/habitat_risk_assessment.rst:92]: Oh, fun! I didn't know that, thanks.

**Review by dcdenu4 — COMMENTED** [review]
Hey @emlys , this looks pretty good to me. I did notice that in the archived html version there is now an `index` button in the top right that sits by the `next` button. Clicking the `index` button sends me to a blank page though.

I was also wondering if anything needed to be updated in the InVEST repo makefile? I briefly looked in there and there's mention of creating a User Guide PDF. I'm not sure that actually happens. But just wanted to make sure there wasn't any coupled things we weren't thinking of.

Thanks!

**Comment by emlys** [comment]
> I did notice that in the archived html version there is now an `index` button in the top right that sits by the `next` button. Clicking the `index` button sends me to a blank page though.

Haha I would have never noticed that! Fixed in 1773b9d

> I was also wondering if anything needed to be updated in the InVEST repo makefile? I briefly looked in there and there's mention of creating a User Guide PDF. I'm not sure that actually happens. But just wanted to make sure there wasn't any coupled things we weren't thinking of.

Good point, there isn't a PDF. I took out that comment and tidied up the `make userguide` command over in the apidocs PR: https://github.com/natcap/invest/pull/395/commits/4366ec2e475b60b5b6bf6a85b1eaf5de2c97a1ca

**Review by dcdenu4 — APPROVED** [review]
Great! Approved.

**Pull request merged**